packages/sync/src/modules/Callables.php

```diff
@@ -371,24 +390,28 @@ public function should_send_callable( $callable_checksums, $name, $checksum ) {
 	 * @access public
 	 */
 	public function maybe_sync_callables() {
+
+		$callables = $this->get_all_callables();
 		if ( ! apply_filters( 'jetpack_check_and_send_callables', false ) ) {
-			if ( ! is_admin() || Settings::is_doing_cron() ) {
-				return;
+			if ( ! is_admin() ) {
+				// If we're not an admin and we're not doing cron, don't sync anything.
```

Comment:
@gititon the `is_admin()` function doesn't check the current user, it checks if it's an admin request (i.e., to an url under `/wp-admin`)